### ISSUE TITLE
Unable to query organization(s) using GraphQL API 

### ISSUE THREAD

**Issue by WJDigby** [SecurityRiskAdvisors/VECTR#124]
It appears that it is not possible to use the GraphQL API to query the "organization" object. Queries to "organization" or "organizations" fail with a `FieldUndefined ValidationError` (see below).

In addition, according to the schema reference none of the other object types return organization(s), so I don't see a workaround to obtain organization information by querying other objects.

Example error using "organization."

**Reqeust**:
```
query {
  organization(db:"TARGET_DB"){
    nodes {
      id
    }
  }
}
```
**Response**:
```
{
  "data": null,
  "errors": [
    {
      "message": "Validation error of type FieldUndefined: Field 'organization' in type 'Query' is undefined @ 'organization'"...
```
https://docs.vectr.io/GraphQL-API/schema/organization.doc.html

Thank you.

**Comment by carlvonderheid** [comment]
Thanks for this.  This is the same problem that is happening in #123 , and will be fixed in the next release

**Comment by carlvonderheid** [comment]
This is actually a separate issue from #123, which was fixed in 6.2.2.  We will fix this in a future release.

**Comment by carlvonderheid** [comment]
Fixed in 7.1.1
